src/comment.ts

```diff
@@ -65,18 +79,30 @@ export const composeCommentsForUsers = (matchingRules: (MatchingRule & { blobURL
       [] as Mention[]
     );
 
-    comments = [...comments, commentTemplate([...new Set(mentions)])];
+    // Since combined comments may originate from multiple rules/teams, we use _combined as the key
+    //  to this comment by convention.
+    comments = {
+      ...comments,
+      _combined: commentTemplate([...new Set(mentions)]),
```

Comment:
can we make this a constant? It was hard for me to realized this was the key you are using the decorate the hidden comment in the comments. Also any reason for calling it `_combined`?  I was thinking that we could just add `combined-comment` or something alike

Reply:
intention here with the leading underscore is to make it clear that this key is not user-defined (i.e. not a path to a herald rule) but instead it's a constant provided by use-herald-action

I'll pull it out as a constant, but I'm less comfortable making it `combined-comment` because it's probably more likely someone out there could define a rule in a file called `combined-comment` than `_combined`

Reply:
alright, I understand. thanks for the feedback. let's move forward with your choice